src/wizards/deployToOpenShift/deployToOpenShift.ts

```diff
@@ -0,0 +1,65 @@
+import { Uri, window } from "vscode";
```

Comment:
If you try to deploy a Quarkus project, but aren't connected to a cluster, then the error message is not friendly.

Reply:
I don't seem to get any error message, but the dialog to push doesn't show up if there is no cluster connected.

Reply:
If you create the component when the cluster is not connected, you do not get validation on the project field, so it could be easy to get the config wrong. It might make sense to only allow deploying to OpenShift through this command when vscode-openshift is able to connect to a cluster.

Reply:
I implemented a hack where it runs `kubectl api-resources` to see if the computer is connected to a cluster. If not connected, then it doesn't try to create the component.